client/components/document.vue

```diff
@@ -12,38 +12,38 @@
       <v-container fluid class="app-layout__users">
         <v-layout row wrap justify-start align-content-start>
           <v-flex class="app-layout__user">
-            <v-btn flat icon style="border-color: #9fa8da;">
```

Comment:
Why removing flat? I think it is not needed, but still, why decided to remove it?